crates/wasmtime/src/runtime/vm/instance/allocator/wasmfx_allocator.rs

```diff
@@ -0,0 +1,261 @@
+// WasmFX fiber stack allocators
+//
+// * on_demand: allocates memory lazily
+// * pooling: preallocates a chunk of memory eagerly
+//
+
+use anyhow::Result;
+use wasmtime_continuations::WasmFXConfig;
+
+cfg_if::cfg_if! {
+    if #[cfg(feature = "wasmfx_baseline")] {
+        use wasmtime_fiber as fiber_impl;
+    } else {
+        use crate::vm::fibre as fiber_impl;
+    }
+}
+
+pub type FiberStack = fiber_impl::FiberStack;
+
+#[cfg(not(feature = "wasmfx_pooling_allocator"))]
```

Comment:
From a quick test, it seems like things still compile with and without `--features wasmfx_pooling_allocator` if this module as well as `wasmfx_pooling` are enabled unconditionally.

Reply:
Undone. It is necessary to pass the `nostd` checks.